### PR TITLE
Fixed Issue #342 (GridPaint screen don't work properly on Firefox)

### PR DESCRIPTION
**Pull request by s-mag**

### CODE REVIEW

File: activities/Gridpaint.activity/js/gridpaint.js [hunk 1]
```diff
@@ -119,21 +119,43 @@ function handleButton(y){
...
-	ctx.scale(scale, scale);
-	for(var i=0;i<shapes.length;i++){	
-		var low = hexdigit(i&0xf);
-		var mid = hexdigit((i>>4)&0xf);
-		var high = hexdigit((i>>8)&0xf);
-		ctx.fillStyle = '#'+high+'F'+mid+'F'+low+'F';
-		shapePath(i, ctx, true);
-	  ctx.fill();		
+var useragent = navigator.userAgent.toLowerCase();
+if (useragent.indexOf('chrome') == -1) {
```
Comment (llaske) [inline activities/Gridpaint.activity/js/gridpaint.js]: Hmmm. Not a big fan of this idea to differentiate the whole process.

Comment (s-mag) [reply activities/Gridpaint.activity/js/gridpaint.js]: Can I know how did you calculate the value for translate (140,20) So that I can calculate common.

Comment (llaske) [reply activities/Gridpaint.activity/js/gridpaint.js]: Unfortunately I don't know. Was there from the beginning.

**Comment by llaske** [comment]
Fixed in https://github.com/llaske/sugarizer/commit/99ed20d71c4f18635efacf0d61b2da383fc28b76

**Pull request closed without merging**